ggshield/hmsl/utils.py

```diff
@@ -18,6 +19,8 @@
 
 logger = logging.getLogger(__name__)
 
+TOKEN_PATH = "{}/hmsl_token".format(get_cache_dir())
```

Comment:
f-strings are more concise:  
```suggestion
TOKEN_PATH = f"{get_cache_dir()}/hmsl_token"
```